docs/source/worked-examples-openstack.rst

```diff
@@ -0,0 +1,112 @@
+Worked Examples: Using Calico-based OpenStack
+=============================================
+
+Once you've installed OpenStack (see :doc:`openstack`) and configured its
+networking (see :doc:`opens-external-conn`) you'll want to let your users start
+playing with your brand new cloud.
+
+Before you do, it will be helpful to have a few worked examples for common
+deployment scenarios. In particular, this will make it easy for you to set up
+topologies and examine their connectivity to try to get an understanding of the
+way Calico networks behave.
+
+This document contains several sections, each of which demonstrates a
+representative deployment model for a kind of workload. The early examples are
+simple, containing small numbers of machines, scaling to the later examples
+that demonstrate quite complex arrangements. Each example contains diagrams
+that represent a connectivity graph, to add clarity to the textual
+descriptions.
+
+Example 1: Development Machine
+------------------------------
+
+In this example, a user wants to spin up a machine to use as a Linux
+development environment. This user is relatively simple: they want a GUI and
+SSH access, but relatively little else.
```

Comment:
I think you should remove the second use of "relatively" in this sentence. It feels a little clunky.